packages/gatsby-plugin-search/src/components/SearchHit.js

```diff
@@ -56,3 +56,61 @@ export default function SearchHit({
     </SearchTeaser>
   );
 }
+
+SearchHitContact.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  image: PropTypes.objectOf([PropTypes.string, PropTypes.number]),
```

Comment:
Får ett fel på denna rad: `Failed prop type: Property `image` of component `SearchHitContact` has invalid PropType notation inside objectOf.`

Reply:
Detta har jag nu löst.